subsystems/networking/networkmanager_linux.go

```diff
@@ -702,6 +702,41 @@ func (n *Networking) backgroundLoop(ctx context.Context, scanChan chan<- bool) {
 	}
 }
 
+func (n *Networking) processUserInput(userInput userInput) bool {
```

Comment:
what does the bool return value represent? the function name is not very descriptive